doc/neps/nep-0052-python-api-cleanup.rst

```diff
@@ -209,15 +212,15 @@ functionality groupings. Also by "mainstream" and special-purpose namespaces:
     numpy.char
     numpy.distutils
     numpy.ma
+    numpy.matlib
```

Comment:
AFAIR you mentioned that `matlib` and `matrixlib` might need stay, but I'm happy to remove them.

`matlib` is deprecated since 1.19.0 (!)
`matrixlib` has a note added 6. years ago that it shouldn't be used:
```
.. note:: It is no longer recommended to use this class, even for linear
          algebra. Instead use regular arrays. The class may be removed
          in the future.
```

I would argue that might be a good timing to drop them. Can I remove them?

Reply:
`np.matrix` and `np.asmatrix` have to stay. `np.bmat` is also in the main namespace. So there is nothing of interest in the `matrixlib` module, I think you can open a PR to remove it.

The `matlib` submodule has more unique functions and a code search for `np.matlib` gives a couple of thousand hits, so that may have to stay indeed for backwards compatibility reasons for now.

Reply:
Sure! I updated the list.